addons/mail/static/src/js/chat_manager.js

```diff
@@ -506,6 +507,8 @@ function on_toggle_star_notification (data) {
             chat_manager.bus.trigger('update_message', message);
         }
     });
+    starred_counter = _.where(messages, {is_starred: true}).length;
```

Comment:
this is wrong.  Let me try to give you a simple example.  Imagine that you have 10 messages starred, but only 1 of them is in your inbox.  Now, you go to your inbox and you will see "10" in the starred channel.  Now, if you refresh the page, the web client will only loads the message in your inbox, since it is your current location.  It means that in the 'messages' list, there is only 1 message that is starred.  So, after the next toggle star notification, discuss will display a totally wrong number (1) instead of the actual number of starred messages